File: android/src/com/dontflymoney/viewhelper/AfterTextWatcher.java
package com.dontflymoney.viewhelper;

import android.text.Editable;
import android.text.TextWatcher;

public abstract class AfterTextWatcher implements TextWatcher {
	
	public abstract void textChanged(String text);
	
	public void afterTextChanged(Editable s) {
    	textChanged(s.toString());
    }
    public void beforeTextChanged(CharSequence s, int start, int count, int after){}
    public void onTextChanged(CharSequence s, int start, int before, int count){}
}

File: android/DFM/src/main/kotlin/com/darakeon/dfm/uiHelpers/watchers/AfterTextWatcher.kt
package com.darakeon.dfm.uiHelpers.watchers

import android.text.Editable
import android.text.TextWatcher

abstract class AfterTextWatcher : TextWatcher {

	abstract fun textChanged(text: String)

	override fun afterTextChanged(s: Editable) {
		textChanged(s.toString())
	}

	override fun beforeTextChanged(s: CharSequence, start: Int, count: Int, after: Int) {}
	override fun onTextChanged(s: CharSequence, start: Int, before: Int, count: Int) {}
}
